packages/landing/src/components/daterange.tsx

```diff
@@ -49,6 +52,12 @@ export class DateRange extends Component {
     this.scheduleUpdateConfig();
   };
 
+  componentDidMount(): void {
+    Config.map.updateFromUrl();
```

Comment:
we should listen for updates so if people update the url this should update.

Reply:
What event do you think is good for tracking url updates.

Reply:
I thought we had something like a on PopState or something somewhere?